lib/commands/find.js

```diff
@@ -16,55 +35,39 @@ helpers.findElOrEls = async function (strategy, selector, mult, context = '') {
     throw new Error("Must provide a selector when finding elements");
   }
 
-  let params = {
+  this.params = {
```

Comment:
I'm not terribly happy about making the parameters for a method be stored on the driver object. Can `params` not remain a local variable that gets passed to `doFindElementOrEls`?